.github/workflows/deploy.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Install supported toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2023-04-10
+          toolchain: nightly-2024-01-19
```

Comment:
let's pin to no specific version but `nightly` directly please

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1305/commits/969a411a23b1cea27856a14cd0a064efb70d1321